src/fragments/start/getting-started/reactnative/integrate.mdx

```diff
@@ -0,0 +1,411 @@
+
+### Querying for Todos
+
+Now that we have a way to add Todo items, we need a way to list them out.
+
+We will use the `useEffect()` hook of **TodoList** component to list the todo items. `useEffect()` can be used to initialize variables on the first render of the component. Add the below `useEffect()` in the **TodoList** component right after state initialization.
+
+```js
+  useEffect(() => {
+    const onQuery = async () => {
+      const todos_list = await DataStore.query(Todo);
+      setTodos(todos_list);
+    };
+
+    // load todos on first render
+    onQuery();
+
+  }, []);
+```
+You can find more about useEffect hook [here](https://reactjs.org/docs/hooks-effect.html).
+
+If you restart your app now, you should see your added todos start to show up on the list! However, if you add more entries, they don’t show up until you refresh the app. Let’s wire up the real-time data observation next.
+
+### Observing updates to Todos
+
+Amplify DataStore provides a way to subscribe to data updates to your model. We will use this to listen for change events and update our list accordingly.
+
+Inside our `useEffect()` hook, we will add a `subscription` to data updates.
+
+```js
+  useEffect(() => {
+
+    //function to query the todolist and store the todolist in state. 
+    const onQuery = async () => {
+      const todos_list = await DataStore.query(Todo);
+      setTodos(todos_list);
+    };
+
+    //load todos on first render 
+    onQuery();
+
+    //subscribe to data updates and fetch the todos using onQuery().
+    //msg variable is the updated todo item and can be used for further customizations.
+    const subscription = DataStore.observe(Todo).subscribe((msg) => {
+      //onQuery will be called each time a data update subscription is received.
+      //onQuery will update the todos list with the latest data/
+      onQuery();
+    });
+
+    //unsubscribe to data updates so that we don’t introduce a memory leak.
+    return function cleanup() {
+      subscription.unsubscribe();
+    }
+  }, []);
+```
+
+If you restart your app now, you should see that newly added todos will also start showing up on the list. The items look like they can be check off and marked as completed, but when pressed, they don’t seem to do anything right now. Let’s learn how to update existing data.
+
+### Updating a Todo
+
+Updating an existing data entry looks a lot like creating a new one. It’s important to note, however, that models in DataStore are *immutable*. So, to update a record you must use a model’s `copyOf` function rather than manipulating its properties directly.
+
+Update the `setComplete` function in the `TodoList` component.
+
+```js
+  async function setComplete(updateValue, todo) {
+    //update the todo item with updateValue
```

Comment:
Nit: Please audit comments to make sure they are consistent i.e.
* Either all or none have whitespace after the //
* Either all or none have the first letter capitalized

Reply:
I have update the comments to make sure they are consistent.